mix.exs

```diff
@@ -24,7 +24,8 @@ defmodule Phoenix.LiveView.MixProject do
     [
       {:phoenix, "~> 1.4.2"},
       {:phoenix_html, "~> 2.13"},
-      {:jason, "~> 1.0", optional: true}
+      {:jason, "~> 1.0", optional: true},
+      {:ex_doc, "~> 0.19.3", runtime: false}
```

Comment:
@joshprice could you add a `only: :dev` here? :)

Reply:
Done!